npm/Fable.Import.Electron.fs

```diff
@@ -516,10 +516,10 @@ module Electron =
         | Normal | Separator | Submenu | Checkbox | Radio
 
     and [<StringEnum>] MenuItemRole =
-        | Undo | Redo | Cut | Copy | Paste | Selectall | Minimize | Close
+        | Undo | Redo | Cut | Copy | Paste | Pasteandmatchstyle | Selectall | Delete | Minimize | Close | Quit | Reload | Forcereload | Toggledevtools | Togglefullscreen | Resetzoom | Zoomin | Zoomout | EditMenu | WindowMenu
```

Comment:
Would it make sense to change `Pasteandmatchstyle` to `PasteAndMatchStyle` and add the `CompiledName` attribute, like [React Native](https://github.com/fable-compiler/fable-react/blob/20efb756b41be784159e2b590726acbedd17d7f9/src/react-native/Fable.Helpers.ReactNative.fs#L25) does?